src/base/static/components/organisms/site-header.js

```diff
@@ -349,6 +364,17 @@ SiteHeader.propTypes = {
   currentUser: PropTypes.object,
   isLeftSidebarExpanded: PropTypes.bool.isRequired,
   languageCode: PropTypes.string.isRequired,
+  mapConfig: PropTypes.shape({
```

Comment:
FYI for future reference, we can store the prop type in the duck. See example here:

https://github.com/mapseed/platform/blob/245d4c683706ef62c98e7bbf6b52db538da19d14/src/base/static/state/ducks/forms-config.js#L12-L22